README.md

```diff
@@ -516,6 +520,63 @@ Reply as usual whilst re-adding the token to your original endpoint `/`
 reply().state('token', token, { path: '/' }).redirect('/wherever');
 ```
 
+## *How do I support users with JS disabled*
+
+An issue can arise when supporting users with JavaScript disabled when JWTs are too large to pass on query strings.
+
+With JS disabled, tokens cannot be added to headers by using redirects from OAuth providers in to the consuming service.
+
+Cloud providers will place limitations on URI lengths
+
+OAuth services may not always sit on a sibling subdomain of the protected service negating the use of a secure cookie
+
+The only way to pass a token in this case is to use either an HTML form with the token in a hidden field and a button with instructions for users to press the button if they have JS disabled and some JS that will submit the form automatically if it is enabled
+
+To configure `hapi-auth-jwt` to support this scenario, you will need to adapt the following sample configuration
+```js
+server.auth.strategy('jwt', 'jwt', {
+      key: 'NeverShareYourSecret',
```

Comment:
that reminds me, we _really_ need to advise people to use Environment Variables for the JWT signing key ... 💭